conf/modules.config

```diff
@@ -749,6 +760,7 @@ if (!params.skip_alignment && params.aligner == 'star_salmon') {
     if (!params.skip_qc & !params.skip_deseq2_qc) {
         process {
             withName: 'DESEQ2_QC_STAR_SALMON' {
+                ext.prefix = { "deseq2" }
```

Comment:
```suggestion
                ext.prefix = "deseq2"
```

Reply:
This was intentional, I thought it would be less error-prone for users to change. But I can remove.